### ISSUE TITLE
Tellefeil på kamper

### ISSUE THREAD

**Issue by havremunken** [RosenborgSupporterSoftware/staut#5]
Nettavisen rapporterer 21 401
STAut rapporterer 21 387

Et avvik på 14 er uholdbart...! ;)

Så spørs det - klarer vi å finne ut hvilke 14 vi mangler?

**Comment by vemundo** [comment]
Hehe, når vi er så nære går det jo faktisk an å studere tallene litt og
kanskje oppdage noe.

Vemund

2015-06-27 20:06 GMT+02:00 Rune Jacobsen notifications@github.com:

**Comment by larsjaas** [comment]
> On 27 Jun 2015, at 20:06, Rune Jacobsen notifications@github.com wrote:
> Nettavisen rapporterer 21 401
> STAut rapporterer 21 387
> 
> Et avvik på 14 er uholdbart...! ;)
> 
> Kanskje ubenyttede reserver telles som tilskuere? ;)

  Lars J

**Comment by nilsgs** [comment]
En start kan være å se på alle koder som har 14 eller mindre salg.

**Comment by havremunken** [comment]
Lars: Hahaha ;)

Yep, jeg har filer tatt i pausen / etter kampen, og disse bør forhåpentligvis kunne fortelle oss noe. 

Jeg er ikke helt enig med meg selv enda, men noe jeg har tenkt hadde vært kult var noe ala "SQL for BS-XML", i.e. en mulighet til å gjøre enkle spørringer mot XML-filene (inntil alle dataene er i databasen). Jeg har jo et enkelt status-query i command line app'en nå, men tenker litt mer generelt, noe som hadde vært anvendbart til dette. Noe som ville gitt oss mulighet til å spørre etter noe ala "koder som ikke betyr solgt/sesongkort, gruppert på kode med antall av hver". Ikke utenkelig at en av kodene dukker opp akkurat 14 ganger. 

Men så virker det også som om utsolgte kamper er litt spesialcaser. Vi får bare se hva vi klarer å finne ut.

Noen måter å legge inn filer som ikke er bilder her som vedlegg? Ville ikke ha xml-fila på første forsøk.

Edit: Dårlig med attachment-muligheter foreløpig. La ut en fil som jeg hentet etter mfk-kampen her: 

https://dl.dropboxusercontent.com/u/7958042/RBK-mfk-270615-2003.xml

**Comment by havremunken** [comment]
Vi bommet med 594 på kvartfinalen mot Mjøndalen. Det er tross alt et ganske heftig avvik. Hvorfor har vi disse avvikene i cup og ecup, men ikke serie?

Det store spørsmålet er vel om dette er 594 som faktisk er i filene fra BillettService. Hvis det er noe som klubben "sitter på" og ikke sender til BS, så er det vel lite å få gjort med. Men ville ikke det være dumt, gitt at man potensielt kunne ende med dobbeltbooking om plassene ikke er markert hos BS? Eller er dette seter som uansett ikke er lagt ut for salg?

Er litt usikker på hvordan jeg skal gå frem for å finne ut av dette. En start hadde kanskje vært å legge til rette for å få alle setestatusene inn i SQL så man kan gjøre litt mer omfattende spørringer.

**Comment by vemundo** [comment]
I den database-skjema diskusjonen vi hadde snakket vi vel om å sette inn status/kode for hvert enkelt sete ved første oppdatering for en ny kamp, og så sette inn bare endringer etter det for seter som endrer status. Eller var det ikke slik?

**Comment by havremunken** [comment]
Jo, det stemmer med hvordan jeg husker det ihvertfall. Og så f.eks. lage et view som gir oss siste status for hvert sete. Det ville ha funka, jeg bare tenkte på å komme igang med noe kjapt - det blir kanskje litt mer omfattende å implementere den totale modellen. Ja, jeg er lat/arbeidssky, det er ikke noe mer enn det. ;)

Men klart, hvis jeg begynner med bare setene, så vil det jo være greit å bare legge til det andre etterpå. Databasen er fortsatt litt "bruk og kast" her, det er trivielt å slette den og lese inn dataene på nytt uansett.

**Comment by havremunken** [comment]
Info fra gisp! i kamptråden mot Stabæk:

Kan noen sjekke kode på de ca fire nederste radene på wb, nest ytterst på Coop-trib. Står tomme foreløpig. Maskotene?
